README.md

```diff
@@ -18,27 +18,51 @@ CONOPS/Rules:
 * [2016 CONOPS/Rules](https://drive.google.com/open?id=0BySpWXvmBM4JUnhyaGN0NW5OR1k)
 
 ### Building and Installing
-Well... seeing as there isn't actually any source in the repository yet, this isn't actually possible at the moment
-
-When we have actually implemented stuff the following will probably apply
 
 **Build system**: CMake to set up build environment then whatever CMake gives you (depends on platform, make for Linux, etc.)
 
 **Dependencies**: 
     [OpenCV 3.0](http://opencv.org/downloads.html)
     [Boost 1.58](http://www.boost.org/users/download/)
-    [DeckLink](http://www.blackmagicdesign.com/support/sdks)
+    [DeckLink](http://www.blackmagicdesign.com/support/sdks) _(optional)_
+    [ZBar] (http://zbar.sourceforge.net/download.html) 
 
 ####Installation Instructions:
-1. Clone the Project
-2. Install dependencies
+1. Install the dependencies listed above. This will vary based on your system. 
+2. Clone the Project
+
+   `git clone https://github.com/UWARG/computer-vision.git`
+
+   `cd computer-vision`
+
 3. Use CMake to setup the project from the CMake list
+
+   `mkdir build`
```

Comment:
Made concise. Build script and build script options need to be written into the docs.uwarg.com website.